homeassistant/components/application_credentials/__init__.py

```diff
@@ -0,0 +1,213 @@
+"""The Application Credentials integration.
+
+This integration provides APIs for managing local OAuth credentials on behalf of other
+integrations. The preferred approach for all OAuth integrations is to use the cloud
+account linking service, and this is the alternative for integrations that can't use
```

Comment:
This isn't clear, since we don't implement cloud account linking on its own. We support a local app by default and cloud account linking is added after that.

Reply:
Removed.